File: plugins/devkit/testData/inspections/getFamilyNameViolation/NotViolatedByField.java
import com.intellij.codeInspection.QuickFix;

class MyQuickFix implements QuickFix {

  String someField;

  public String getName() {
    return "some name";
  };

  public String getFamilyName() {
    return someField + getName() + "123";
  };


}

File: plugins/devkit/devkit-kotlin-tests/testData/inspections/getFamilyNameViolation/NotViolatedByField.kt
import com.intellij.codeInspection.ProblemDescriptor
import com.intellij.codeInspection.QuickFix
import com.intellij.openapi.project.Project

class NotViolatedByField : QuickFix<ProblemDescriptor> {
  private var someField: String? = null

  override fun getName(): String {
    return "some name"
  }

  override fun getFamilyName(): String {
    return someField + name + "123"
  }

  override fun applyFix(project: Project, descriptor: ProblemDescriptor) {
    // any
  }
}
